lib/kits/paged-publication/viewer.ts

```diff
@@ -310,6 +311,28 @@ class Viewer extends MicroEvent {
         this.processHotspotQueue();
     }
 
+    applyPageDecorations(pageDecorations) {
```

Comment:
This method should follow the behavior of `Viewer#applyHotspots` and render the decorations into the viewer.

If more advanced integrators(for example our SDK's `paged-publication` component) want to render the decorations outside of the viewer, they should simply not use `Bootstrapper#applyPageDecorations` and instead use the result of `Bootstrapper#fetchPageDecorations` to render their page decorations as they see fit.